mkdocs.yaml

```
@@ -177,7 +177,9 @@ nav:
     - Quickstart: 'integration/quickstart.md'
   - APIs:
     - Library API: 'api.md'
-    - Web API: 'https://api.near.ai/openapi.json'
+    - Web API Docs: 'https://api.near.ai/redoc'
+    - Interactive API: 'https://api.near.ai/docs'
+    - OpenAPI Specs: 'https://api.near.ai/openapi.json'
```

```suggestion
    - Hub Standard API: 'https://docs.near.ai/api/hub/standard'
    - Hub Interactive API: 'https://api.near.ai/api/hub/interactive'
```

I would prefer something closer to this, rather than having both `api.near.ai/docs` AND `docs.near.ai/api`

---

URL paths for the auto-generated docs can be customized with:

```py
FastAPI(docs_url="/api/hub/interactive", redoc_url="/api/hub/standard")
```

To-do: we need to decide the best paths for these 2 pages.